lib/util/ui_helper.dart

```diff
@@ -1120,74 +1120,106 @@ class UIHelper {
-                              borderColor: AppColor.primaryBlack,
-                              textColor: AppColor.primaryBlack,
-                              backgroundColor: AppColor.auSuperTeal,
-                            ),
-                            const SizedBox(
-                              height: 15,
-                            )
-                          ],
+                        if (actionButtonOnTap != null)
+                          Column(
+                            children: [
+                              AuSecondaryButton(
+                                text: actionButton ?? "",
+                                onPressed: actionButtonOnTap,
+                                borderColor: AppColor.primaryBlack,
+                                textColor: AppColor.primaryBlack,
+                                backgroundColor: AppColor.auSuperTeal,
+                              ),
+                              const SizedBox(
+                                height: 15,
+                              )
+                            ],
+                          ),
+                        AuSecondaryButton(
+                          text: exitButton ?? "close".tr(),
+                          onPressed: exitButtonOnTap ??
+                              () {
+                                Navigator.pop(context);
+                              },
+                          borderColor: AppColor.primaryBlack,
+                          textColor: AppColor.primaryBlack,
+                          backgroundColor: AppColor.auSuperTeal,
                         ),
-                      AuSecondaryButton(
-                        text: exitButton ?? "close".tr(),
-                        onPressed: exitButtonOnTap ??
-                            () {
-                              Navigator.pop(context);
-                            },
-                        borderColor: AppColor.primaryBlack,
-                        textColor: AppColor.primaryBlack,
-                        backgroundColor: AppColor.auSuperTeal,
-                      ),
-                    ],
+                      ],
+                    ),
                   ),
                 ),
               ),
-            ),
-          );
-        });
+            ));
+  }
+
+  static Future<dynamic> showCenterEmptySheet(BuildContext context,
+      {required Widget content}) async {
+    UIHelper.hideInfoDialog(context);
+    return await showCupertinoModalPopup(
+        context: context,
+        builder: (context) => Scaffold(
+              body: Stack(
+                children: [
+                  GestureDetector(
+                    child: Container(
+                      color: AppColor.primaryBlack.withOpacity(0.8),
+                    ),
+                    onTap: () {
+                      Navigator.pop(context);
+                    },
+                  ),
+                  Center(
```

Comment:
can we use padding to wrap center, so we don't need to use Spacer()?

Reply:
it didn't work